src/main/resources/site/macros/staticVisualization/staticVisualization.ts

```diff
@@ -6,7 +6,29 @@ const {
   preview
 } = __non_webpack_require__('../../parts/staticVisualization/staticVisualization')
 
+const {
+  preview: dividerControllerPreview
+} = __non_webpack_require__('../../parts/divider/divider')
+
+const {
+  renderError
+} = __non_webpack_require__('/lib/ssb/error/error')
+
 exports.macro = (context: MacroContext): Response | React4xpResponse => {
-  const config: StaticVisualizationConfig = context.params
-  return preview(context, config.staticVisualizationContent)
+  try {
+    const divider: Response = dividerControllerPreview(context, {
+      dark: false
+    })
+
+    const config: StaticVisualizationConfig = context.params
+    const staticVisualization: Response = preview(context, config.staticVisualizationContent)
+
+    if (staticVisualization.status && staticVisualization.status !== 200) throw new Error(`Highchart with id ${config.staticVisualizationContent} missing`)
```

Comment:
Should error message be "staticVisualization with id [...]" instead of "Highchart with id [...]"?